app/webhook.go

```diff
@@ -705,17 +705,34 @@ func (a *App) CreateCommandWebhook(commandId string, args *model.CommandArgs) (*
 		ParentId:  args.ParentId,
 	}
 
-	return a.Srv().Store.CommandWebhook().Save(hook)
+	savedHook, err := a.Srv().Store.CommandWebhook().Save(hook)
+	if err != nil {
```

Comment:
We also need a check for `case errors.As(err, &appErr):` because the IsValid method returns an AppError, so we would want to propagate it upwards as-is.